sonar-project.properties

```diff
@@ -4,7 +4,7 @@ sonar.projectDescription=SonarJS rules for ESLint
 
 # current version in development
 # should be bigger than the one in package.json
-sonar.projectVersion=0.6.0
+sonar.projectVersion=0.12.0
```

Comment:
should it be `0.13.0` keeping in mind the comment? Or should we drop the comment? As we are using "maven"-like style with package.json holding the version being under development